assignments/prototypes.js

```diff
@@ -15,13 +15,32 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+function GameObject (attributes) {
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+};
 
+GameObject.prototype.destroy = function() {
+  return `${this.name} was removed from the game.`
+};
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function CharacterStats(PersonalStats) {
+  GameObject.call(this, PersonalStats);
+  this.healthPoints = PersonalStats.healthPoints;
+}
+
+CharacterStats.prototype = Object.create(GameObject.prototype);
+
+CharacterStats.prototype.takeDamage = function() {
+  return `${GameObject.name} took damage`;
```

Comment:
```suggestion
  return `${this.name} took damage`;
```

Reply:
the 'this' keyword will point to name property of the object that takeDame fn is being called on